db/projects/kyber-network/project.yaml

```diff
@@ -0,0 +1,29 @@
+name: Kyber Network
```

Comment:
Please add `alt_names` with "KyberNetwork", its very common - for example on etherscan:
https://etherscan.io/token/0xdd974d5c2e2928dea5f71b9825b8b646686bd200

Reply:
KyberNetwork was set as alt_name of asset